view/adminhtml/templates/modal.phtml

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Aligent\Pci4Compatibility\Block\Adminhtml\Modal;
+use Magento\Framework\Escaper;
+
+/**
+ * @var Escaper $escaper
+ * @var Modal $block
+ */
+
+$sessionTimeout = $block->getModalPopupTimeout();
+
+?>
+
+<div id="session-expire-warning-modal">
+    <p style="font-size:48px;color:#FF0000">&#x26A0;</p>
+    <p>Your admin session will expire in one minute, please save your changes or refresh the page.</p>
```

Comment:
Do we want to actually give the option to extend the session asynchronously? It would be more user-friendly.

Reply:
The trouble with just reloading the page is that the user is likely to lose whatever changes they've got. We should be able to execute an AJAX request that can renew their session, but remain on the page they're currently on.
Obviously, we'd require testing to make sure that whatever page they're editing remains saveable (no form key issues, etc.)